thirdparty/fetch-thirdparty-deps.ps1

```diff
@@ -1,3 +1,5 @@
+New-Item -ItemType Directory -Force -Path windows
```

Comment:
Can you give me the full line? I can't test myself for the correct syntax and AppVeyor is slow to respond.